core/src/test/java/hudson/XmlFileTest.java

```diff
@@ -19,17 +19,29 @@
 class XmlFileTest {
 
     @Test
-    void canReadXml1_0Test() throws IOException {
+    public void canReadXml1_0WhenFileExists() throws IOException {
```

Comment:
JUnit 5 does not require that the test method is public.  Please leave this line unchanged.
```suggestion
    void canReadXml1_0Test() throws IOException {
```